src/DKIMMailServiceProvider.php

```diff
@@ -35,7 +35,10 @@ private function publishesConfig(): void
     protected function registerIlluminateMailer()
     {
         $this->app->singleton('mail.manager', static function (Application $app) {
-            return new MailManager($app);
+            if (config('dkim.domain') !== null) {
+                return new MailManager($app);
+            }
+            return new \Illuminate\Mail\MailManager($app);
```

Comment:
```suggestion
            if (config('dkim.domain') === null) {
                return new \Illuminate\Mail\MailManager($app);
            }
            return new MailManager($app);
```

Reply:
I think this version would even be better.

Reply:
sure, also fine :+1: